app/code/Magento/Sales/Plugin/Model/ResourceModel/Order/OrderGridCollectionFilter.php

```diff
@@ -55,7 +55,7 @@ public function aroundAddFieldToFilter(
                 }
             }
 
-            $fieldName = $subject->getConnection()->quoteIdentifier($field);
+            $fieldName = "main_table.".$subject->getConnection()->quoteIdentifier($field);
```

Comment:
Could you please format code with spaces
```suggestion
            $fieldName = "main_table." . $subject->getConnection()->quoteIdentifier($field);
```